content/docs/configuration-system/content.md

```diff
@@ -43,10 +55,8 @@ $ knoxite config init
 ```
 
 This will leave us with an empty configuration file:
-```json
-{
-    "repositories": {}
-}
+```toml
+[repositories]
 ```
 
 By default knoxite will use the OS dependant standard paths for this file:
```

Comment:
depend**e**nt dominates in American English

Reply:
using the same version as in our code comment